OrbitQt/GgpClientTests.cpp

```diff
@@ -0,0 +1,73 @@
+
```

Comment:
Is this compiled anywhere?

Reply:
see my comment above. You caught it

Reply:
Ah I see. "save locally" means you remove it? That's at least what I would suggest in this case.

Reply:
yes, already done